packages/cli/src/lib/snapshot/reportViewer.ts

```diff
@@ -42,43 +66,62 @@ export class ReportViewer {
   }
 
   // TODO: Change logic once SRC-4448 is complete
-  private printTableSection(row: {
-    resourceName: string;
-    operations: ResourceSnapshotsReportOperationModel;
-  }) {
+  private printTableSection(
+    row: {
+      resourceName: string;
+      operations: ResourceSnapshotsReportOperationModel;
+    },
+    indentation = 3
+  ) {
     const resourceType = this.prettyPrintResourceName(row.resourceName);
-    let output = `   ${resourceType}\n`;
+    let output = `${''.padStart(indentation)}${resourceType}\n`;
 
-    if (row.operations.resourcesCreated > 0) {
+    if (
+      this.operationsToDisplay.resourcesCreated &&
+      row.operations.resourcesCreated > 0
+    ) {
       output += `${ReportViewer.styles.green(
-        '+'
-      )}   ${ReportViewer.styles.green(
+        '+'.padEnd(indentation + 1)
```

Comment:
Same as above, there's quite a bit of repetition to build the output.

Suggestion: Just a small util method that accept a color, a sign and an indentation could make this easier to read ?